src/linpred.jl

```diff
@@ -113,14 +119,24 @@ end
 Base.cholfact{T}(p::SparsePredChol{T}) = copy(p.chol)
 Base.cholfact!{T}(p::SparsePredChol{T}) = p.chol
 
+"""
+Extract the estimates of the coefficients in the model
+"""
 coef(x::LinPred) = x.beta0
 coef(x::LinPredModel) = coef(x.pp)
 
+"""
+Degrees of freedom for residuals, when meaningful
```

Comment:
No, this returns the *number of residual degrees of freedom* (I agree the name is confusing).